.github/workflows/ci-cd.yml

```
@@ -136,13 +136,12 @@ jobs:
   notify-slack:
     runs-on: ubuntu-latest
     needs: [deploy-staging, deploy-production]
-    if: always()
+    if: always() && secrets.SLACK_WEBHOOK != ''
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)